src/controllers/user.controllers.js

```diff
@@ -10,6 +10,14 @@ export const createUser = async (data) => {
   const { error } = registerValidation(data);
   if (error) return { status: 'invalid', message: error.details[0].message };
 
+  const users = await User.find({ email: data.email });
+
+  const userExist = users.filter((user) => {
+    if (user.statusUser === 1) return user;
```

Comment:
Would probably be best if this `1` was stored as a const like `const USER_ACTIVE = 1` so that `USER_ACTIVE` is referenced instead.